woocommerce/wc-admin
# Add welcome modal when coming from Calypso

Fixes https://github.com/Automattic/wp-calypso/issues/48420

The intent of https://github.com/Automattic/wp-calypso/issues/48420 is to show a welcome modal when entering WooCommerce Admin from Calypso. https://github.com/automattic/wp-calypso/issues/48420 adds `&from-calypso` to the URL to WCA, and this PR handles that URL param to display the welcome modal.

### Screenshots
![image](https://user-images.githubusercontent.com/224531/103834641-bb8d8b00-50cf-11eb-9aee-84ed43881c64.png)
*edit* Updated with final copy

### Detailed test instructions:
- Starting with a fresh store (or by deleting the `woocommerce_task_list_welcome_modal_dismissed` option), visit `/wp-admin/admin.php?page=wc-admin`. You should see the standard welcome modal.
- Add `&from-calypso` to the URL. You should see the Calypso welcome modal.
- Dismiss the modal
- Refresh the page. The modal should not reappear.
- Remove the `&from-calypso` from the URL. The standard welcome modal should not reappear.
